SpywareFilter/sections/specific.txt

```diff
@@ -108,6 +108,7 @@ the-ans.jp##.side-reccomend
 ||laurenhi.com/app/Eclog/
 /lcslog*.js$domain=playentry.org|modoo.at|~pulmuri.modoo.at
 /clickcr*.js$domain=naver.com|modoo.at|~help.naver.com
+/google-analyticator/*-tracking.min.js
```

Comment:
I see `/google-analyticator/*` in EP. I guess can be added as `/google-analyticator/*$script` (will change in master)

Reply:
Gotta fix them all. I have checked the whole website from top to bottom.